packit/distgit.py

```diff
@@ -191,10 +211,12 @@ def sync_files(self, upstream_project: LocalProject) -> None:
         """
         logger.debug("about to sync files %s", self.files_to_sync)
         for fi in self.files_to_sync:
+            # TODO: fi can be dir
             fi = fi[1:] if fi.startswith("/") else fi
             src = os.path.join(upstream_project.working_dir, fi)
             if os.path.exists(src):
                 logger.info("syncing %s", src)
                 shutil.copy2(src, self.local_project.working_dir)
             else:
-                logger.debug("not found %s (no sync)", src)
+                # TODO: is this enough?
+                logger.warning("not found %s (no sync)", src)
```

Comment:
I feel it more like filter (like `.gitignore`) and not much care about not matched items. We can support some wildcards in the future as well. But do it like you feel it needs to be done.

Reply:
this is not gitignore, it's actually exact opposite: users ask us to synchronise files and if we can't find those files, it means that we can't do our job: so it's definitely at least a warning; I wouldn't mind throwing an exc here to be honest

Reply:
yes, wildcard support is also something we'll add sooner or later; also support for directories

Reply:
OK, I will stay with the warning.